ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue

```diff
@@ -41,7 +48,7 @@
                   class="pt-4 pr-2"
                   label="First Name"
                   v-model="submittingParty.personName.first"
-                  :rules="[]"
+                  :rules="firstNameRule"
```

Comment:
Sorry, nitpicking here, but rules are plural not singular 😁 Could we update the rule names?